simp_le.py

```diff
@@ -391,6 +391,33 @@ def save(self, data):  # pylint: disable=missing-docstring
         return super(FullChainFile, self).save(self.Data(
             key=data.key, cert=None, chain=([data.cert] + data.chain)))
 
+@IOPlugin.register(path='full.der', typ=OpenSSL.crypto.FILETYPE_ASN1)
+@IOPlugin.register(path='full.pem', typ=OpenSSL.crypto.FILETYPE_PEM)
+class FullFile(OpenSSLIOPlugin):
+    """Certificate and privatekey and chain lugin."""
```

Comment:
typo: lugin -> plugin, maybe be clearer about the order of the entries in the file?

    """"Key plus full certificate chain file plugin.""""